node/test/dialog.spec.ts

```diff
@@ -34,7 +34,7 @@ describe('dialog', async () => {
   let app: Express;
   let mockAxios: MockAxios;
   let mockNextRandom: sinon.SinonStub<number[]>;
-  const allScenarios: DialogScenario[] = await findAllScenarios();
+  let allScenarios: DialogScenario[] = await findAllScenarios();
```

Comment:
this may have been result of a merge, but why would we make this `let` and not `const`?